app.js

```diff
@@ -1,11 +1,58 @@
 /* Imports */
+import { getBabies, getSigns } from './fetch-utils.js';
+import { renderBabies, renderAstroSigns } from './render-utils.js';
 
 /* Get DOM Elements */
-
+const babyList = document.querySelector('.baby-list');
+const formEl = document.querySelector('form');
+const dropdown = document.querySelector('.dropdown');
+const selectEl = document.querySelector('select');
+    
 /* State */
+let babies = [];
+let astroSigns = [];
 
 /* Events */
+// addEventListener on load of window
+window.addEventListener('load', async () => {
+   //request babies
+    const response = await getBabies();
+    babies = response;
+
+    //calling the display function thats below 
+    displayBabies();
+
+    const signs = await getSigns();
+
+    //looping the signs and appended the new option el to the dom
+    for (let sign of signs) {
+        const optionEl = document.createElement('option');
```

Comment:
this logic looks great. when you're ready to refactor, you can put this in your renderAstroSigns() and make a call to that function within your loop here.